package.json

```diff
@@ -10,12 +10,13 @@
   "scripts": {
     "build-bundle": "scripts/build-bundle.sh",
     "cover": "npm run jest -- --coverage",
-    "dev": "gulp docs-generate && webpack-dev-server --inline --hot --content-base src/docs/",
+    "dev": "webpack-dev-server --inline --hot --content-base src/docs/",
```

Comment:
is this useful? why not just make the `peek` task `dev` instead?

Reply:
whoops i didn't mean to leave that in there, that was for my own local testing only. will remove